examples/Cargo.toml

```diff
@@ -1,20 +1,20 @@
 [package]
 name = "iroh-examples"
-version = "0.1.3"
-edition = "2021"
+version.workspace = true
+edition.workspace = true
 authors = ["dignifiedquire <me@dignifiedquire.com>"]
-license = "Apache-2.0/MIT"
-repository = "https://github.com/n0-computer/iroh"
+license.workspace = true
+repository.workspace = true
 description = "Examples"
-rust-version = "1.65"
+rust-version.workspace = true
 publish = false
 
 [dependencies]
 anyhow = { version = "1", features = ["backtrace"] }
-bytes = "1.1.0"
+bytes.workspace = true
 bytesize = "1.1.0"
-clap = { version = "4.0.9", features = ["derive"] }
-futures = "0.3.21"
+clap.workspace = true
+futures.workspace = true
```

Comment:
i realise this example still needs cleaning up.  i'll do this once there's approval in principle.  this might also race the iroh-embed pr - so one of them will need fixing up anyway.  and examples are touched by that pr